app/src/components/dialog/ViewFileWithDetailsDialog.tsx

```diff
@@ -53,6 +56,8 @@ const useStyles = makeStyles((theme: Theme) => ({
 }));
 
 export interface IViewFileWithDetailsDialogProps {
+  projectId: number;
+  surveyId?: number;
```

Comment:
We dont need to worry about this right now necessarily, but if we think passing both IDs and then checking them is a bit annoying, we can always split this into 2 separate components, one for project and one for survey. 

Most of the internal code is probably re-usable, and could be moved to a separate generic component. The event handling functions could then be passed in as needed, with different function calls for the project and survey endpoints, etc.

Reply:
Will leave for now, as its a simple solution. Tech debt would be to clean up these components